src/components/GithubLink.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Text, View, TouchableHighlight } from 'react-native';
+import styles from '../screens/ProfileStyleSheet';
+
+export default class componentName extends Component {
```

Comment:
Hi @YvesIraguha 
Is it possible to make `componentName` more descriptive so that anyone reading your code can know the purpose/ function of this Component?

Reply:
Definitely, I am going to change it to a more descriptive name. Thanks